### ISSUE TITLE
Unable to get property 'push' of undefined or null reference

### ISSUE THREAD

**Issue by rolandoldengarm** [AzureAD/azure-activedirectory-library-for-js#303]
I've created a basic app with some routes, and I'm initializing the adal provider.
When opening the app, I get the following error.
````
HTML1300: Navigation occurred.
index.html
>>> Office.initialize()
TypeError: Unable to get property 'push' of undefined or null reference
   at m (https://secure.aadcdn.microsoftonline-p.com/lib/1.0.10/js/adal-angular.min.js:2:3331)
   at Scope.prototype.$broadcast (https://localhost:44308/Scripts/angular.js:17552:15)
   at updateRoute (https://code.angularjs.org/1.2.25/angular-route.js:497:9)
   at Scope.prototype.$broadcast (https://localhost:44308/Scripts/angular.js:17552:15)
   at afterLocationChange (https://localhost:44308/Scripts/angular.js:13394:7)
   at Anonymous function (https://localhost:44308/Scripts/angular.js:13380:13)
   at Scope.prototype.$eval (https://localhost:44308/Scripts/angular.js:17229:9)
   at Scope.prototype.$digest (https://localhost:44308/Scripts/angular.js:17045:15)
   at Scope.prototype.$apply (https://localhost:44308/Scripts/angular.js:17337:13)
   at bootstrapApply (https://localhost:44308/Scripts/angular.js:1749:9)
Agave.HostCall.IssueCall
Agave.HostCall.ReceiveResponse
````
I have no idea how to debug this, because there's not much in the app. The default view only contains some HTML...

I'm bootstrapping manually as it is an Office Addin, however that does not make a difference.

**Comment by tushargupta51** [comment]
@rolandoldengarm Its a known issue: https://github.com/AzureAD/azure-activedirectory-library-for-js/issues/282

This will be fixed in 1.0.11

For now, add this to your adal.init configuration:
`anonymousEndpoints: []`

Closing this as dup of #282. If you still have any issues, please comment on #282.

**Comment by rolandoldengarm** [comment]
Thanks, that fixed it.
